packages/http-server/src/http-server.ts

```diff
@@ -163,10 +150,26 @@ export class HttpServer {
     return this._listening;
   }
 
+  public get server(): http.Server | https.Server {
+    return this._server;
+  }
+
   /**
    * Address of the HTTP / HTTPS server
    */
   public get address(): AddressInfo | undefined {
     return this._listening ? this._address : undefined;
   }
 }
+
+/**
+ * Default implementation of HttpServerFactory
+ */
+export class DefaultHttpServerFactory {
```

Comment:
Do we have to use a class here? Can we use a plain function instead?

```ts
export function defaultHttpServerFactory(
  requestListener: RequestListener,
  serverOptions?: HttpServerOptions,
): HttpServer {
  return new DefaultHttpServer(requestListener, serverOptions);
}
```

See also [KISS principle](https://en.wikipedia.org/wiki/KISS_principle).